python/ql/src/Variables/MultiplyDefined.ql

```diff
@@ -67,5 +67,5 @@ where
   forall(Name el | el = asgn1.getParentNode().(Tuple).getAnElt() | multiply_defined(el, _, _)) and
   not uninteresting_definition(asgn1)
 select asgn1,
-  "This assignment to '" + v.getId() +
-    "' is unnecessary as it is redefined $@ before this value is used.", asgn2 as t, "here"
+  "This assignment to '" + v.getId() + "' is unnecessary as $@ before this value is used.", asgn2,
+  "it is redefined"
```

Comment:
```suggestion
  "This assignment to '" + v.getId() + "' is unnecessary as it is $@ before this value is used.", asgn2,
  "redefined"
```
This seems more consistent with a/an being pushed out of the interpolated part in many of the other messages.